chains/53935/chain-information.json

```diff
@@ -1,22 +1,27 @@
 {
   "chainName": "DFK Chain",
-  "description": "The vision for the DFK Chain on the Defi Kingdoms subnet is to become the go-to location for community members and projects to launch GameFi and other blockchain gaming experiences in conjunction with the primary offering of DeFi Kingdoms: Crystalvale.",
+  "description": "",
```

Comment:
I just noticed that we removed all of these chain descriptions.  Not sure how that happened but we need to put them back.